### PR TITLE
rgb() syntax changes and minor issue with class name solved

### LINKED ISSUE #2706: try it confusion
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

### What specific section or headline is this issue about?

try it

### What information was incorrect, unhelpful, or incomplete?

"<button class="favorite styled"

### What did you expect to see?

"<button class="styled" 

### Do you have any supporting links, references, or citations?

The favorite is very confusing

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/button`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/button/index.md
* Last commit: https://github.com/mdn/content/commit/aa5dfde3e95f9d0f2cd73e6d639313e4e44a5418
* Document last modified: 2023-07-27T06:35:38.000Z

</details>

### PR DESCRIPTION
**Pull request by ShubhamOulkar**
In this PR , 

- [x] removed unused class name.
- [x] updated rgb() syntax from css file 

Related issues and pull requests

[issue mdn/content#30844](https://github.com/mdn/content/issues/30844)

Closes [issue mdn/interactive-examples#2706](https://github.com/mdn/interactive-examples/issues/2706)

### CODE REVIEW

**Comment by hamishwillee** [comment]
FYI, best to do fixes for separate examples as separate PRs because your fix for one might be right and the other not.

**Comment by ShubhamOulkar** [comment]
this PR include only 1 example

File: live-examples/html-examples/input/css/button.css [hunk 1]
```diff
@@ -7,24 +7,24 @@
   color: #fff;
   text-shadow: 1px 1px 1px #000;
   border-radius: 10px;
-  background-color: rgba(220, 0, 0, 1);
+  background-color: rgb(220 0 0 /100%);
```
Comment (dipikabh) [inline live-examples/html-examples/input/css/button.css]: Add a space after the `/` in all the instances.

File: live-examples/html-examples/input/css/button.css [hunk 2]
```diff
@@ -7,24 +7,24 @@
...
   background-image: linear-gradient(
     to top left,
-    rgba(0, 0, 0, 0.2),
-    rgba(0, 0, 0, 0.2) 30%,
-    rgba(0, 0, 0, 0)
+    rgb(0 0 0 /0.2),
+    rgb(0 0 0 /0.2) 30%,
+    rgb(0 0 0 /0)
```
Comment (dipikabh) [inline live-examples/html-examples/input/css/button.css]: The numbers after the `/` can be expressed as percentages here.

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, @ShubhamOulkar. The changes look good overall. Just a few minor updates needed.

**Comment by ShubhamOulkar** [comment]
@dipikabh , writing a % sign for 0 value (alpha values) is necessary for technical documentation? we avoid writing in large code bases. 
[PR #31497](https://github.com/mdn/content/pull/31497)

**Comment by dipikabh** [comment]
Hello @ShubhamOulkar, thanks for checking.
You are correct that we don't specify a unit when the value is `0`. However, in the case of `rgb()`, it would be better to retain the `%` for the sake of consistency of format. `rgb(0 0 0 / 0%)` expresses opacity as a percentage and `rgb(0 0 0 / 0)` expresses opacity as a decimal number and both are valid. 

We use percentages in all other `rgb()` instances, like `rgb(255 255 255 / 30%)`, so using `rgb(0 0 0 / 0%)` aligns with this format. This approach ensures uniformity across all our RGB values, enhancing clarity and standardization in our code style.

File: live-examples/html-examples/input/css/button.css [hunk 3]
```diff
@@ -7,24 +7,24 @@
...
-  background-color: rgba(220, 0, 0, 1);
+  background-color: rgb(220 0 0 / 100%);
   background-image: linear-gradient(
     to top left,
-    rgba(0, 0, 0, 0.2),
-    rgba(0, 0, 0, 0.2) 30%,
-    rgba(0, 0, 0, 0)
+    rgb(0 0 0 / 20%),
+    rgb(0 0 0 / 20%) 30%,
+    rgb(0 0 0 / 20%)
```
Comment (dipikabh) [inline live-examples/html-examples/input/css/button.css]: Slight nit here, the original opacity was 0
```suggestion
    rgb(0 0 0 / 0%)
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Review by dipikabh — APPROVED** [review]
@ShubhamOulkar, thank you for the `rgb()` updates and fixing the class issue! 🎉

**Pull request merged**